docs/reports/commitment-discounts.md

```diff
@@ -0,0 +1,158 @@
+![Status: Not started](https://img.shields.io/badge/status-not%20started-red) &nbsp;<sup>→</sup>&nbsp;
+[![Go to issue](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/1)](https://github.com/microsoft/cloud-hubs/issues/1)
+
+# FinOps hubs Commitment discounts report
+
+The **Commitment discounts report** summarizes existing and potential savings from commitment-based discounts, like reservations and savings plans. This report enables you to:
+
+- Review Azure Hybrid Benefit usage.
+- Identify and resolve any under-utilized commitments (aka utilization).
+- Identify opportunity to save with more commitment-based discounts (aka coverage).
+- Determine which resources used commitment-based discounts (aka chargeback).
+- Summarize cost savings from commitment-based discounts.
+
+> 🚩 **Important**<br>FinOps hubs uses amortized costs. Amortization breaks reservation and savings plan purchases down and allocates costs to the resources that received the benefit. Due to this, amortized costs will not show purchase costs and will not match your invoice. Please use Cost Management to review invoice charges.
```

Comment:
Is there a link we can include to explain more about amortized cost, I'm concerned very new users will still not understand. Also can we link "Cost Management" to the actual portal landing page?